source/sass/patterns/molecules/inline-list.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../mixins/typography";
+
+.inline-list {
+  @include list-style-none();
+  overflow-y: hidden;
+}
+
+.inline-list__item {
+  display: inline-block;
+  font-family: $font-secondary;
+
+  .inline-list--small & {
+    @include set-font-size-and-line-height(scale(-1));
+  }
+
+  .inline-list & {
+    @include comma-separator();
+  }
+
+  .inline-list--alternative-separators & {
+    &:after {
```

Comment:
This doesn't override language specific options.

Reply:
🤦‍♂